### PR TITLE
CTA for free trials

### PR DESCRIPTION
**Pull request by lyleschemmerling**
image

### CODE REVIEW

**Review by dcharles8 — CHANGES_REQUESTED** [review]
Left a comment regarding the URL. Besides that, click testing all looked good to me.

File: astro/src/components/cta/FreeTrialCTA.astro [hunk 1]
```diff
@@ -0,0 +1,24 @@
+---
+---
+<div id="container" class="px-5 h-full flex flex-col justify-evenly space-y-4 py-8">
+  <h3 class="text-xl leading-7 font-black">Your <span class="text-orange-400">Auth</span>, Your Way - We Handle the Rest</h3>
+  <p class="text-base font-medium">Start 30-day <span class="font-bold text-green-300">Free</span> Trial.</p>
+  <a href="https://account-local.fusionauth.io/account/free-trial/start?source=site" class="text-center w-full bg-orange-400 rounded px-4 py-2.5 text-sm font-bold uppercase">Start Free Trial</a>
```
[the remark this answers is not in the document]
Comment (dcharles8) [reply astro/src/components/cta/FreeTrialCTA.astro]: Updated

**Review by dcharles8 — APPROVED** [review]

**Review by bhalsey — CHANGES_REQUESTED** [review]
CTA text isn't readable in light mode

**Comment by lyleschemmerling** [comment]
image

**Review by bhalsey — APPROVED** [review]

**Pull request merged**